docs/dev/Lua API.rst

```diff
@@ -2595,10 +2595,12 @@ The ``textures`` module solves this problem by providing a stable handle instead
 raw ``texpos``. When we need to draw a particular tile, we can look up the current
 ``texpos`` value via the handle.
 
-* ``loadTileset(file, tile_px_w, tile_px_h)``
+* ``loadTileset(file, tile_px_w, tile_px_h, reserved?)``
 
   Loads a tileset from the image ``file`` with give tile dimensions in pixels. The
   image will be sliced in row major order. Returns an array of ``TexposHandle``.
+  ``reserved`` is optional boolean argument, which indicates texpos range.
+  ``true`` - reserved, ``false`` - dynamic (default).
```

Comment:
maybe add a few sentences to the introduction section (line 2567) explaining what the reserved range is. Also, we should document what happens if the reserved range is already full.